cmd/monaco/delete/delete.go

```diff
@@ -40,18 +41,18 @@ func Delete(ctx context.Context, environments manifest.EnvironmentDefinitionsByN
 	for _, env := range environments {
 		ctx := context.WithValue(ctx, log.CtxKeyEnv{}, log.CtxValEnv{Name: env.Name, Group: env.Group})
 		if containsPlatformTypes(entriesToDelete) && !env.HasPlatformCredentials() {
-			log.WarnContext(ctx, "Delete file contains Dynatrace Platform specific types, but no platform credentials are defined for environment %q - Dynatrace Platform configurations won't be deleted.", env.Name)
+			slog.WarnContext(ctx, "Delete file contains entries referencing platform configurations, but environment is missing platform credentials. These configurations won't be deleted.")
 		}
 
 		clientSet, err := client.CreateClientSet(ctx, env.URL.Value, env.Auth)
 		if err != nil {
 			return fmt.Errorf("failed to create API client for environment %q due to the following error: %w", env.Name, err)
 		}
 
-		log.InfoContext(ctx, "Deleting configs for environment %q...", env.Name)
+		slog.InfoContext(ctx, "Deleting configs...")
```

Comment:
I noticed the environment name was deleted. Is there no value in knowing which environment we are deleting? wdyt?

Reply:
Actually, it is already added via [line 42](https://github.com/Dynatrace/dynatrace-configuration-as-code/pull/1996/files/611666fa8169dd572a585ee0a01a0260ff86c3f3#diff-865abfef2995909e444fbd2c57d7ab6136fee53d35d58f81b4d200c58b832d02R42). Removing it here stops it being added twice